src/scripts/mapDiseaseOutbreakToAlerts.ts

```diff
@@ -0,0 +1,251 @@
+        args: {},
+        handler: async () => {
+            if (!process.env.VITE_DHIS2_BASE_URL)
+                throw new Error("VITE_DHIS2_BASE_URL must be set in the .env file");
+
+            if (!process.env.VITE_DHIS2_AUTH)
+                throw new Error("VITE_DHIS2_AUTH must be set in the .env file");
+
+            const username = process.env.VITE_DHIS2_AUTH.split(":")[0] ?? "";
+            const password = process.env.VITE_DHIS2_AUTH.split(":")[1] ?? "";
+
+            if (username === "" || password === "") {
+                throw new Error("VITE_DHIS2_AUTH must be in the format 'username:password'");
+            }
+
+            const envVars = {
+                url: process.env.VITE_DHIS2_BASE_URL,
+                auth: {
+                    username: username,
+                    password: password,
+                },
+            };
+
+            const api = getD2ApiFromArgs(envVars);
+            const alertRepository = new AlertD2Repository(api);
+            const alertSyncRepository = new AlertSyncDataStoreRepository(api);
+            const notificationRepository = new NotificationD2Repository(api);
+            const optionsRepository = new OptionsD2Repository(api);
+
+            const notifyWatchStaffUseCase = new NotifyWatchStaffUseCase(notificationRepository);
+
+            return Future.joinObj({
+                alertTrackedEntities: alertRepository.getTrackedEntitiesByTEACode({
+                    program: RTSL_ZEBRA_ALERTS_PROGRAM_ID,
+                    orgUnit: RTSL_ZEBRA_ORG_UNIT_ID,
+                    ouMode: "DESCENDANTS",
+                }),
+                hazardTypes: optionsRepository.getAllHazardTypes(),
+                suspectedDiseases: optionsRepository.getAllSuspectedDiseases(),
+            }).run(
+                ({ alertTrackedEntities, hazardTypes, suspectedDiseases }) => {
+                    const alertsWithNoEventId = _(alertTrackedEntities)
+                        .compactMap(trackedEntity => {
+                            const nationalEventId = alertRepository.getTEAttributeById(
+                                trackedEntity,
+                                RTSL_ZEBRA_ALERTS_NATIONAL_DISEASE_OUTBREAK_EVENT_ID_TEA_ID
+                            );
+                            const hazardType = alertRepository.getTEAttributeById(
+                                trackedEntity,
+                                RTSL_ZEBRA_ALERTS_EVENT_TYPE_TEA_ID
+                            );
+                            const diseaseType = alertRepository.getTEAttributeById(
+                                trackedEntity,
+                                RTSL_ZEBRA_ALERTS_DISEASE_TEA_ID
+                            );
+
+                            const alertData: AlertData = {
+                                alert: trackedEntity,
+                                attribute: diseaseType ?? hazardType,
+                                dataSource: diseaseType ? "IBS" : "EBS",
```

Comment:
Get the changes from development because the codes for DataSource has changed in the optionSets in the instance. Sorry because this is something we need to fix, we should not be coupled to dhis2 in layers other than data layer.